install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
I'm concerned about automatically calling this function, and calling it on `nvm use`. Typically, shell completions are explicitly opt-in by users.

In addition, I think this needs to be installed in *only* the `NVM_PROFILE` file above - ie, only the one(s) in which `nvm.sh` is sourced - not in all of them. If we're going to put `bash_completion` in up to 3 files, then `nvm.sh` should be sourced in all 3.